docs/get-response-codes.md

```diff
@@ -0,0 +1,36 @@
+# GetResponseCodes
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-01
+
+## Output Message
+
```

Comment:
Lets use the new template that was decided for the help files.

Reply:
cant find the link where this was documented. Please check with Tejaswi

Reply:
[https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#how-to-write-a-new-validation-rule-using-typescript](https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#how-to-write-a-new-validation-rule-using-typescript)